README.md

```diff
@@ -49,6 +49,7 @@ The builder will run the same build as `@angular-devkit/build-angular:browser` d
 Options:
  - All the `@angular-devkit/build-angular:browser` options
  - `webpackConfigPath`: path to the extra webpack configuration, defaults to `webpack.config.js`
+ - `mergeStrategy`: webpack config merge strategy, strategies can be `append | prepend | replace`. Defaults to append
```

Comment:
Need to explain more about strategies (what does it mean and how each one works).